BraintreeModels/src/main/java/com/braintreepayments/api/Authorization.java

```diff
@@ -1,15 +1,13 @@
 package com.braintreepayments.api;
 
-import android.os.Parcel;
-import android.os.Parcelable;
 import android.text.TextUtils;
 
 import androidx.annotation.Nullable;
 
 /**
  * Generic base class for Braintree authorization
  */
-abstract class Authorization implements Parcelable {
```

Comment:
Is removing `Parcelable` part of the removing the `throws` from the `BraintreeClient` constructor? Or just an additional thing we wanted to do that is in this PR?

Reply:
When we created the new `InvalidAuthorization` class that extends from `Authorization` we realized that `Parcelable` wasn't needed since these classes are no longer merchant-facing. Instead of adding it unnecessarily to the new class, we decided to remove it from `Authorization` all together. We can add a note to the PR description about this for clarity.

Reply:
Gotcha - thanks for clarifying!